.vscode/launch.json

```diff
@@ -41,6 +41,18 @@
       "outFiles": [
         "${workspaceFolder}/packages/app-studio-toolkit/dist/**/*.js"
       ]
+    },
+    {
+      "name": "Run Validation",
```

Comment:
Use the **exact** name of the vscode extension.
This is because we have many launch configurations here so we should start
to be more strict.

Reply:
Done